test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -400,6 +438,9 @@ private class TestController
             [ValueProviderMetadata]
             public string ValueBinderMarkedProperty { get; set; }
 
+            [CustomBindingSource]
```

Comment:
good catch : yeah you are right it doesnt come into play, in a normal (e2e or integartion scenario) this would be required however this is just for completeness ( it will be confusing to see a property which is not marked, being bound). I will update the test to use BindingSource.Custom.